android/src/org/ppsspp/ppsspp/SizeManager.java

```diff
@@ -107,7 +115,11 @@ public void surfaceChanged(SurfaceHolder holder, int format, int width, int heig
 		NativeApp.backbufferResize(width, height, format);
 		updateDisplayMeasurements();
 
-		activity.notifySurface(holder.getSurface());
+		if (!paused) {
+			activity.notifySurface(holder.getSurface());
+		} else {
+			Log.i(TAG, "Skipping notifySurface while paused");
```

Comment:
This can't cause it to have an outdated surface, can it?

-[Unknown]

Reply:
It does appear to be OK because we always get a surface notification after the next resume, which is when we want it.

I believe spurious surfaceChanged events after pause are a bug that seems to have been fixed around Android 12.